bitnami/ghost/templates/deployment.yaml

```diff
@@ -116,89 +129,90 @@ spec:
             {{- include "common.tplvalues.render" ( dict "value" .Values.extraEnvVars "context" $ ) | nindent 12 }}
             {{- end }}
-          {{- if or .Values.extraEnvVarsConfigMap .Values.extraEnvVarsSecret }}
+          {{- if or .Values.extraEnvVarsCM .Values.extraEnvVarsSecret }}
           envFrom:
-            {{- if .Values.extraEnvVarsConfigMap }}
+            {{- if .Values.extraEnvVarsCM }}
             - configMapRef:
-                name: {{ include "common.tplvalues.render" ( dict "value" .Values.extraEnvVarsConfigMap "context" $ ) }}
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsCM "context" $) }}
             {{- end }}
             {{- if .Values.extraEnvVarsSecret }}
             - secretRef:
-                name: {{ include "common.tplvalues.render" ( dict "value" .Values.extraEnvVarsSecret "context" $ ) }}
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
             {{- end }}
           {{- end }}
           ports:
+            {{- if .Values.ghostEnableHttps }}
+            - name: https
+              containerPort: {{ .Values.containerPorts.https }}
+              protocol: TCP
+            {{- else }}
             - name: http
-              containerPort: 2368
+              containerPort: {{ .Values.containerPorts.http }}
+              protocol: TCP
+            {{- end }}
           {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-              path: {{ .Values.ghostPath }}
+              path: /
               port: http
-              httpHeaders:
-                - name: Host
-                  value: {{ include "ghost.host" . | quote }}
-                {{- if eq .Values.ghostProtocol "https" }}
-                - name: X-Forwarded-Proto
-                  value: https
-                {{- end }}
             initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
             timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
-            successThreshold: {{ .Values.livenessProbe.successThreshold }}
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
           {{- else if .Values.customLivenessProbe }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customLivenessProbe "context" $) | nindent 12 }}
           {{- end }}
           {{- if .Values.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
-              path: {{ .Values.ghostPath }}
+              path: /
               port: http
-              httpHeaders:
-                - name: Host
-                  value: {{ include "ghost.host" . | quote }}
-                {{- if eq .Values.ghostProtocol "https" }}
```

Comment:
Same here, if we drop https support inside the container we should document it and explain why

Reply:
We're not dropping it. We have now `ghostEnableHttps` that you can set to `true/false` and, Ghost will automatically choose `http` or `https` depending on that.